v20.2/admin-ui-overview.md

```diff
@@ -24,6 +24,7 @@ Area | Description
 [Replication Dashboard](admin-ui-replication-dashboard.html) | Metrics about how data is replicated across the cluster, e.g., range status, replicas per store, and replica quiescence.
 [Changefeeds Dashboard](admin-ui-cdc-dashboard.html) | Metrics about the [changefeeds](stream-data-out-of-cockroachdb-using-changefeeds.html) created across your cluster.
 [Databases](admin-ui-databases-page.html) | Details about the system and user databases in the cluster.
+[Sessions](admin-ui-sessions-page.html) | Details about open sessions on the cluster.
```

Comment:
nit: "on" disagrees with the "in" that we have in the same phrase one line above. IMO, "in" sounds better as well.